specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json

```diff
@@ -4803,12 +4803,6 @@
           },
           "204": {
             "description": "No Content"
-          },
-          "default": {
```

Comment:
Why is the default error response removed?

Reply:
This is the only integration account artifact that has the default response. This appears to make the SDK return a 404 response instead of a CloudException, which is what all the other artifacts return.